tests/automation_framework/data/work_order/work_order_get_result.ini

```diff
@@ -0,0 +1,29 @@
+[DEFAULT]
+method=WorkOrderGetResult
+id=11
+params:{"workOrderId":""}
+
+[18873]
+workOrderId:11b2dd896b3c2198fa6ef0c51988800faa82dbc12010ec710aacce938f87a06p
+
+[18874]
+workOrderId:\$!@**^(
+
+[18875]
+workOrderId:null
+
+[18876]
+workOrderId:"001122"
+
+[18877]
+workOrderId:abcdefghijklmnop
+
+[18878]
+workOrderId:123
+
+[20322]
+workOrderId:remove
+
+[18879]
+workOrderId:remove
```

Comment:
Why do we want to remove "workOrderId". The test is for unknown parameter which we have defined below.

Reply:
In workordergetresult_unknownparameter.json, there was no workOrderId field that's why i removed it.

JSON Params:
"params": {
        "workOrderId123": ""
    }